tests/unit/seq_execution/src/suit_condition_image_match.c

```diff
@@ -178,13 +178,23 @@ void test_seq_execution_condition_image_match_no_image_size(void)
 		.value = seq_cmd,
 		.len = sizeof(seq_cmd),
 	};
+	struct zcbor_string exp_decoded_digest = {
```

Comment:
Looking at the name of this test case it seems that it tests what happens if the parameter "size" is not given, but the size parameter won't be needed anymore - so is this test still useful? Or perhaps it should be renamed?

Reply:
Well, in fact it checks if it is possible to check digest without specifying the size :slightly_smiling_face:

Reply:
Test case removed!